core/src/main/scala/org/apache/spark/ui/jobs/JobProgressListener.scala

```diff
@@ -19,7 +19,7 @@ package org.apache.spark.ui.jobs
 
 import java.util.concurrent.TimeoutException
 
-import scala.collection.mutable.{HashMap, HashSet, LinkedHashMap, ListBuffer}
```

Comment:
Although these other changes are not wrong, I don't think they're related. Best to stick to the core change you're making.